LICENSE.txt

```diff
@@ -0,0 +1,28 @@
+This software is OSI Certified Open Source Software.
+OSI Certified is a certification mark of the Open Source Initiative.
+
+Copyright (c) 2009-2018, Enthought, Inc.
```

Comment:
Thanks so much for this, would you mind changing the copyright statement to the following? We're trying to keep all of our copyright headers uniform.

```
(C) Copyright 2009-2019 Enthought, Inc., Austin, TX
All rights reserved.
```

Reply:
I would be happy to change it.  However, I have looked through all of the enthought packages and none of them use the copyright header you gave.  They all use the copyright header I had originally (which is because I copied it from the other licenses you were using).  Again, if you really want it changed I can do so.  But that would make it inconsistent with the other packages, so I wanted to double-check to make sure that the change is correct before I do it.

Reply:
Thanks for checking that, @toddrme2178. Indeed the new copyright header is what we want going forward, but we do not have to make any sweeping changes of every file, just updating existing copyright headers or adding new ones whenever a file happened to be touched in a PR. Since this is such a small project, I went ahead and added headers to all the existing code a few weeks back. We just happen to be the first to implement it for this project. All of the other ETS projects will catch up slowly but surely.